src/query.coffee

```diff
@@ -32,7 +32,12 @@ class Query extends EventEmitter
       @emit 'error', err
 
   onRowDescription: (msg) ->
-    throw new errors.VerticaError("Cannot handle multi-queries with a callback!") if @callback? && @status?
+    if @status
+      err = new errors.VerticaError("Cannot handle multi-queries with a callback!")
+      if @callback
+        @error = err
+      else
+        @emit 'error', err
```

Comment:
I think it currently works fine if you're not using the callback method, but use an event listener instead. So we probably don't want to emit an error in this case.